### DISCUSSION TITLE
Possible to get barbecue to work with Org files?

### DISCUSSION THREAD

**Discussion by is-ahmed** [utilyre/barbecue.nvim#72] [Q&A]
Hi, so I was wondering if it was possible to get barbecue to work with .org files? Sometimes my org files can get pretty big and I think it would be useful if barbecue was able to tell my where I am in the file.

![image](https://user-images.githubusercontent.com/60582433/222856387-6914bdef-49fd-437c-b818-fbb062905760.png)

I want it to be like sample.org > Header 1 > Header 2 > Header 3

**Comment by utilyre** [comment]
Hello! This is a really good idea, but I personally never gotten into org mode and neorg. So I don't know much about it to make this happen. But feel free to open a feature request issue in case somebody wanted to implement this.
